internal/conf/config.yaml

```
@@ -154,9 +154,22 @@ webserver:
     rotationday: 0        # day of the week for weekly rotation, 0 = Sunday
 
 security:
-  host: ""                   # host and port for autoTLS and authentication
-  autotls: false             # true to enable auto TLS, only host is whitelisted
-  redirecttohttps: false     # true to redirect http to https
+  # host is required for AutoTLS and OAuth providers
+  # Must be a fully qualified domain name (e.g., "birdnet.example.com")
+  # The domain must point to this server's public IP address
+  host: ""
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)